test/system/220-healthcheck.bats

```diff
@@ -273,4 +273,215 @@ Log[-1].Output   | \"Uh-oh on stdout!\\\nUh-oh on stderr!\\\n\"
     done
 }
 
+@test "podman healthcheck --health-max-log-count default value (5)" {
+    local repeat_count=10
+    local msg="Hello, How are you?"
+    local ctrname="c-h-$(safename)"
+    run_podman run -d --name $ctrname   \
+               --health-cmd "echo $msg" \
+               $IMAGE /home/podman/pause
+    cid="$output"
+
+    run_podman inspect $ctrname --format "{{.Config.HealthMaxLogCount}}"
+    is "$output" "5" "HealthMaxLogCount is set to 5"
+
+    for _ in $(seq 1 $repeat_count);
+    do
+        run_podman healthcheck run $ctrname
+        is "$output" "" "output from 'podman healthcheck run'"
+    done
+
+
+    run_podman inspect $ctrname --format "{{.State.Health.Log}}"
+    count=$(grep -o "$msg" <<< "$output" | wc -l)
+    assert "$count" -eq 5
```

Comment:
I don't think this produces nice errors either but I leave to @edsantiago to judge.